plugins/SimulationView/PathSlider.qml

```diff
@@ -47,6 +48,14 @@ Item {
         rangeHandle.width = handle.x - sliderRoot.handleSize
     }
 
+    function normalizeValue(value) {
+        if (value > sliderRoot.maximumValue)
+            return sliderRoot.maximumValue
+        else if (value < sliderRoot.minimumValue)
+            return sliderRoot.minimumValue
+        return value
+    }
```

Comment:
Same here.

Reply:
Done in 1a189ba4f5be0d0a377ddcdd8671e0a59b5d7533